scenarios/classification/21_deployment_on_azure_container_instances.ipynb

```diff
@@ -30,8 +30,8 @@
     "  1. [Workspace retrieval](#workspace)\n",
```

Comment:
<p>[minor] Consider adding a bit of text before this and the next cell to explain what the cell does.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/529/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Done.